test/spec/modules/dataController_spec.js

```diff
@@ -0,0 +1,378 @@
+import {expect} from 'chai';
+import {config} from 'src/config.js';
+import {filterData} from 'modules/dataControllerModule/index.js';
+import * as utils from 'src/utils.js';
+
+const MODULE_NAME = 'dataController';
+let dataControllerConfig;
+const ALL = '*';
+
+function filterEIDs(requestObject) {
```

Comment:
This is still a work in progress I assume - this logic looks like it belongs in the module.

Reply:
This module development is completed. The test contains how the submodule looks like

Reply:
the test is testing test code? that can't be right. I'm missing the point entirely, once the bundle is built this code (and therefore the tests on it) are completely irrelevant no? the module in the actual codebase is an empty shell.